teku/src/main/java/tech/pegasys/teku/cli/options/DepositOptions.java

```diff
@@ -27,11 +28,11 @@
   private Eth1Address eth1DepositContractAddress = null; // Depends on network configuration
 
   @Option(
-      names = {"--eth1-endpoint"},
+      names = {"--eth1-endpoint", "--eth1-endpoints"},
       paramLabel = "<NETWORK>",
-      description = "URL for Eth1 node.",
-      arity = "1")
-  private String eth1Endpoint = null;
+      description = "URL for Eth1 nodes.",
```

Comment:
So that `--eth1-endpoint http://localhost:8545,http://oopsy.local:8545` works:

```suggestion
      description = "URL for Eth1 nodes.",
      split = ",",
```

Reply:
Actually, given that we haven't got the `DepositContract` updated yet to support fallback, that makes it pretty much impossible to test this well, so I think we're probably better off leaving the CLI as accepting only a single argument at this point until we get the internals fully sorted.  Basically toggle off the feature at the CLI level for the moment.  So this field would be a `String` and not a `List` but then `getEth1Endpoints()` method would still return a `List` - it would just never contain more than one item.

We could add a new, hidden `--eth1-endpoints` option to make testing easier if you wanted to and then merge the two options when we're ready to support it, but could also just modify the code for local testing for now.

Sorry that mucks you around again - just that this is a fairly popular feature so I suspect people will discover it if its exposed and that will create confusion.